src/Microsoft.VisualStudio.Editors/PropPages/CodeAnalysisPropPage.resx

```diff
@@ -157,10 +160,13 @@
     <value>NoControl</value>
   </data>
   <data name="NETAnalyzersLinkLabel.Location" type="System.Drawing.Point, System.Drawing">
-    <value>39, 154</value>
+    <value>52, 190</value>
+  </data>
+  <data name="NETAnalyzersLinkLabel.Margin" type="System.Windows.Forms.Padding, System.Windows.Forms">
+    <value>4, 0, 4, 0</value>
   </data>
   <data name="NETAnalyzersLinkLabel.Size" type="System.Drawing.Size, System.Drawing">
-    <value>132, 13</value>
+    <value>175, 17</value>
```

Comment:
Was VS running in high-dpi mode here, or with scaling? I ask because the change in sizes/locations throughout this file is quite significant.

When I run VS in high-DPI mode (the default) the WinForms designer shows a warning message that changing the layout can result in failures on traditional 96-DPI screens. The advice is to restart VS to disable high-DPI mode and use pixel scaling instead, so that VS thinks it's running at 96-DPI.

More information at https://docs.microsoft.com/en-us/visualstudio/designers/disable-dpi-awareness?view=vs-2019

Reply:
Thanks for noticing that, I fixed it in the new commit. 👍